### ISSUE TITLE
Color contrast in "Getting started" dashboard footer links

### ISSUE THREAD

**Issue by BatJan** [umbraco/Umbraco-CMS#7056]
During the accessibility audit made by @dlancaster it was discovered that the color contrast in the footer links is insufficient. Since the markup is generated outside the Umbraco source code we're not able to make PR's for fixing this ourselves, hence this issue 😃 

According to https://contrast-ratio.com/#%23F2FCFC-on-%2300BEC1 the contrast score is 2.2. Ideally it should be 4.5:1 for normal text like this is. More information can be found here https://webaim.org/resources/contrastchecker/

![footer-links](https://user-images.githubusercontent.com/1932158/68187624-69bb1380-ffa7-11e9-80f7-1c727286d5bd.png)

**Comment by nul800sebastiaan** [comment]
Thanks @BatJan, I'll need to get someone from our team to look at it!

**Comment by emma-hq** [comment]
Hey @BatJan 

We have fixed this one [here](https://github.com/umbraco/OnlineMedia.Issues/issues/57) so I'm going to close :)

Emma

**Comment by nul800sebastiaan** [comment]
That link goes to an internal issue tracker.. but you get the idea Jan! 😅
